<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuDialog</class>
 <widget class="QDialog" name="MenuDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>562</width>
    <height>372</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>返回登录</string>
   </property>
  </widget>
  <widget class="QPushButton" name="epyManageBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>210</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>人员管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="proManageBtn">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>210</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>项目管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="aptManageBtn">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>部门管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="qiye_info">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>企业信息</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
